code/src/components/App.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import data from '../data.json'
+import { Header } from './Header.js'
+import { Albums } from './Album.js'
+import { Artists } from './Artists.js'
+import { ReleaseTitle } from './ReleaseTitle.js'
+
+//*Main rendering component
+export const App = () => {
+  return (
+    <div className='section-container'>
+      <Header />
+      <section className='rows'>
+
+      {/*Mapping through all the albums in the json */}
+      {data.albums.items.map((album, i) => {
+        return (
+            <div className='album-container' key={i}>
+            <Albums key={album.id} url={album.images[0].url} name={album.name} />
```

Comment:
`key` attribute is considered as required only in root element inside loop. We don't have to add it anywhere else. This snippet of code is wrong :

```jsx
<div key={someUniqueValue}>
  <h1 key={someUniqueValue}>Heading</h1>
  <p key={someUniqueValue}>Paragraph</p>
  <span key={someUniqueValue}>Span</span>
</div>
```

This snippet of code is correct :

```jsx
 <div key={someUniqueValue}>
  <h1>Heading</h1>
  <p>Paragraph</p>
  <span>Span</span>
</div>
```

Reply:
fixed and thank you for the valuable feedback